src/lib/core/CHIPCallback.h

```diff
@@ -49,9 +49,7 @@ class Cancelable
      */
     Cancelable * mNext;
     Cancelable * mPrev;
-
-    void * mInfoPtr;
-    uint64_t mInfoScalar;
+    alignas(uint64_t) char mInfo[16];
```

Comment:
should we use https://en.cppreference.com/w/cpp/types/max_align_t instead? Likely 64-bit will have the same result however max-align seems to make the purpose clearer.

Reply:
memcpy doesn't depend on alignment anyway so even removing the alignment completely is safe irrespective of the type stored. The only benefit of alignment is the potential for more efficient machine code.